README.md

```diff
@@ -40,30 +40,40 @@ Spoke & Chain shows core Craft CMS features and a fully-configured Craft Commerc
 If you’d like to get Spoke & Chain running in a local environment, we recommend using [DDEV](https://ddev.com):
 
 1. Clone the Spoke & Chain repository to your system.
-    ```zsh
-    git clone git@github.com:craftcms/spoke-and-chain.git spokeandchain && cd spokeandchain
-    ```
-1. Follow Craft related DDEV [installation instructions](https://craftcms.com/knowledge-base/migrating-from-craft-nitro-to-ddev).
-1. Make sure you’ve used `ddev config` to create your environment.
-1. Copy and update your `.env` file:
-    ```zsh
-    cp .env.example .env
-    ```
-1. Restore the initial database
-   ```zsh
-   ddev exec php craft db/restore seed.sql
+   ```sh
+   git clone git@github.com:craftcms/spoke-and-chain.git spokeandchain && cd spokeandchain
    ```
-1. Optionally seed demo data:
-   ```zsh
-   ddev exec php craft demos/seed
+2. Install the Composer dependencies:
+   ```sh
+   composer install
+   ```
```

Comment:
Might want to do `ddev composer install` so we don't have to assume they have composer globally in PATH.